src/applications/edu-benefits/containers/PreSubmitInfo.jsx

```diff
@@ -29,17 +16,12 @@ function PreSubmitNotice({ formData, showError, setPreSubmit }) {
         withholding information or for providing incorrect information. (See 18
         U.S.C. 1001)
       </div>
-      <Checkbox
-        required
-        checked={privacyAgreementAccepted}
-        onValueChange={value => setPreSubmit('privacyAgreementAccepted', value)}
-        name={'privacyAgreementAccepted'}
-        errorMessage={
-          showError && !privacyAgreementAccepted
-            ? 'You must accept the privacy policy before continuing.'
-            : undefined
+      <VaPrivacyAgreement
```

Comment:
va-privacy-agreement on storybook for reference: https://design.va.gov/storybook/?path=/docs/components-va-privacy-agreement--default